lib/seadsa/DsaCallSite.cc

```diff
@@ -134,6 +114,20 @@ DsaCallSite::const_formal_iterator DsaCallSite::formal_end() const {
                                      getCallee()->arg_end());
 }
 
+DsaCallSite::const_formal_iterator
+DsaCallSite::formal_func_begin(const llvm::Function &F) const {
```

Comment:
Comment why this is a method in `DsaCallSite`. It seems to work on a given function `F`, not necessarily either the caller nor the calee that the`DsaCallSite` represents

Reply:
I wasn't sure where else to put it. Would making a FunctionUtils file be more appropriate?

Reply:
if it is just a function and not a method, make it a function. There are lots of examples of functions that return ranges already. Don't make it member of `DsaCallSite`. Since the function is small, you can keep it in a header file.